app/controllers/concerns/foreman_puppet_enc/controller/parameters/config_group.rb

```diff
@@ -0,0 +1,17 @@
+module ForemanPuppetEnc::Controller::Parameters::ConfigGroup
```

Comment:
Not necessary, this exists as
```suggestion
module ForemanPuppetEnc::Parameters::ConfigGroup
```